### PR TITLE
Add Rossmann icon

### PR DESCRIPTION
**Pull request by dirien**
Signed-off-by: Engin Diri <engin.diri@mail.schwarz>

image

**Issue:** #6458 
**Alexa rank:** 33,902
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

Icon -> https://www.rossmann.de/unternehmen/resources/templating-kit/themes/rsmn-corp/5.97.0-2021.08.16.05.53/images/layout/logo.svg

### CODE REVIEW

**Comment by dirien** [comment]
Create a comparison
image

**Review by sachinraja — CHANGES_REQUESTED** [review]
image

**Comment by dirien** [comment]
Hi @sachinraja, thanks for pointing out. Fixed it :)

**Comment by dirien** [comment]
@sachinraja: I used this file -> https://www.rossmann.de/unternehmen/resources/templating-kit/themes/rsmn-corp/5.97.0-2021.08.16.05.53/images/layout/logo.svg

File: _data/simple-icons.json [hunk 1]
```diff
@@ -8557,6 +8557,12 @@
             "source": "https://www.ros.org/press-kit/",
             "guidelines": "https://www.ros.org/press-kit/"
         },
+        {
+            "title": "Rossmann",
+            "hex": "C3002D",
+            "source": "https://www.rossmann.de",
+            "guidelines": "https://www.rossmann.de/unternehmen/presse/media-downloads/logos.html"
```
Comment (sachinraja) [inline _data/simple-icons.json:8564]: Icon is a perfect match, but this page does not seem to exist. It just redirects to https://unternehmen.rossmann.de/presse/mediathek.html which is not a guidelines page.

**Review by sachinraja — CHANGES_REQUESTED** [review]

**Comment by dirien** [comment]
@sachinraja removed the guideline tags. Looks like they changed their HP

**Review by sachinraja — APPROVED** [review]

**Pull request merged**